examples/mnist/export_test.py

```diff
@@ -0,0 +1,64 @@
+import tensorflow as tf
+import tensorflow_hub as hub
+import unittest
+import tempfile
+import shutil
+import os
+from absl.testing import absltest
+import export
+from export import train_and_export
```

Comment:
Please don't import functions, but modules instead:
from examples.mnist import export

https://github.com/google/styleguide/blob/gh-pages/pyguide.md#22-imports

Reply:
Thanks for sharing the link :)